### ISSUE TITLE
ERROR: ld.so: object '/home/username/.local/share/Steam/ubuntu12_32/gameoverlayrenderer.so' from LD_PRELOAD cannot be preloaded (wrong ELF class: ELFCLASS32): ignored.

### ISSUE THREAD

**Issue by GreatBigWhiteWorld** [ValveSoftware/steam-for-linux#4652]
#### Your system information

* Steam client version:1474415843
* Distribution (e.g. Ubuntu): openSUSE TW
* Opted into Steam client beta?: [Yes/No] No
* Have you checked for system updates?: [Yes/No]Yes

#### Please describe your issue in as much detail as possible:
Describe what you _expected_ should happen and what _did_ happen. Please link any large code pastes as a [Github Gist](https://gist.github.com/)

#### Steps for reproducing this issue:

1. launch steam from command line
2. launch dota 2 with "primusrun %command%" parameter
3. game performance is extremely low (20fps), should be 60fps like before.

If I don't use primusrun I still have 20fps so it seems nvidia card is not working?

**Comment by Tele42** [comment]
Hello @GreatBigWhiteWorld,

The title of this issue report is invalid and a harmless warning, please rephrase it to something more productive. Additionally, please include enough detail to make this actionable, like when you started encountering this issue, do you know of any updates between the last time it was behaving properly and when you started experiencing this issue, and what date was the last time you know it was behaving properly.

**Comment by Plagman** [comment]
I would check your NVIDIA driver setup or your primus setup. As @Tele42 said the spew about ld.so ignoring the 32-bit version of the overlay is harmless.
